src/wallets/wallets.js

```diff
@@ -410,8 +429,8 @@ export async function signGroup(info, txnarray) {
     case "MyAlgoConnect": {
       return await signSet(myAlgoConnect, senderAddressObj, txnarray)
     }
+    case "Defly":
```

Comment:
Never seen this syntax before. Is this how you do an "OR" in a case statement? Just making sure Defly is handled here...